lib/aoc.rb

```diff
@@ -44,25 +48,25 @@ def self.next_puzzle_time
   end
 
   def self.release_time(day)
-    Time.new(2023, 12, day, 0, 0, 0, event_timezone)
+    Time.new(year, 12, day, 0, 0, 0, event_timezone)
   end
 
   def self.url(day)
-    "https://adventofcode.com/2023/day/#{day}"
+    "https://adventofcode.com/#{year}/day/#{day}"
   end
 
   # Le Wagon specific times and helpers
 
   def self.lewagon_launch_time
-    Time.new(2023, 11, 17, 8, 30, 0, pilou_timezone)
+    Time.new(year, 11, 19, 8, 30, 0, pilou_timezone)
   end
 
   def self.lewagon_lock_time
-    Time.new(2023, 12, 8, 17, 30, 0, pilou_timezone)
+    Time.new(year, 12, 8, 17, 30, 0, pilou_timezone)
```

Comment:
On utilise encore lock_time tient ?

Reply:
Feature-wise, non plus du tout :
* le Ladder of Insanity n'est plus opt-in/opt-out
* les squads c'est devenu le marché ouvert
* les achievements ne sont plus affichés nulle part (pour le moment)

Ça fait partie des trucs à dégommer dans #586, je me le garde sous le coude pour plus tard.